lib/assessment/code_feature_extractor.py

```diff
@@ -14,7 +13,6 @@
     "rect": ["x", "y", "w", "h"],
     "regularPolygon": ["x", "y", "sides", "size"],
     "rgb": ["r", "g", "b", "a"],
-    "shape": ["x1", "y1", "xn", "yn"],
```

Comment:
Removed this, since it was causing issues in detecting shapes. Does not hamper other CFE functionality.